app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/StringLiterals
**Severity**: INFO
**File**: app/controllers/api/v1/users_controller.rb L14

Prefer single-quoted strings when you don&apos;t need string interpolation or special symbols.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*